daal4py/sklearn/_utils.py

```diff
@@ -411,3 +378,66 @@ def method_wrapper(self, *args, **kwargs):
         return result
 
     return method_wrapper
+
+
+def control_n_jobs(decorated_methods: list):
```

Comment:
I think we should add a few tests.

* Does a class with `n_jobs` in `__init__()` behave as expected
* Does a class without `n_jobs` in `__init__()` behave as expected
* Are the functions in `decorated_methods` modified as expected
* Are functions *not* in `decorated_methods` untouched

Reply:
Decoration check was added in https://github.com/intel/scikit-learn-intelex/pull/1622/commits/25725d8901abdbf3b472cb80846e4471e1f52095